### ISSUE TITLE
Balasana Theme ::  Google Page Speed Tool errors. 

### ISSUE THREAD

**Issue by mdtanjid0** [Automattic/themes#7343]
### Quick summary

With the Balasana theme, it shows an error on Google Page Speed Insights tool `Values assigned to role="" are not valid ARIA roles.`. 

 

### Steps to reproduce

1. Install Balasana Theme: https://wordpress.com/theme/balasana/
2. Add the primary menu. 
3. And check with the Google Page Speed Insights Tool: https://pagespeed.web.dev/analysis
4. And check the error: 
![Screen Shot on 2023-09-06 at 21:41:23](https://github.com/Automattic/themes/assets/54123436/823af8fb-5293-4181-9e11-18c70a290522)

### What you expected to happen

It shouldn't show this error, and the role value should be valid. 

### What actually happened

It shows an error as the role value indicates invalid by Google Page Speed Insights Tool. 

### Context

#6697903-zen

### Platform (Simple, Atomic, or both?)

Simple, Atomic

### Reproducibility

Consistent

### Severity

Some (< 50%)

### Available workarounds?

No but the platform is still usable

**Comment by cuemarie** [comment]
📌 REPRODUCTION RESULTS
- Tested on Simple – Replicated

📌 FINDINGS/SCREENSHOTS/VIDEO
I activated Balasana on a brand new free WordPress.com site, and analyzed it using https://pagespeed.web.dev/

- With a Primary Menu set, running the site through Google Page Speed insights has this ARIA warning appear under the `Desktop` analysis.

![jjUMcn.png](https://github.com/Automattic/themes/assets/27249804/143d4a47-4f05-407d-8087-9a8363cb9404)

```
Failing Elements noted:
div#page > header#masthead > nav#site-navigation > input#toggle
<input type="checkbox" role="button" aria-haspopup="true" id="toggle" class="hide-visually">
```

📌 ACTIONS
- Retitling
- Adding to Bug Scrub Planning board
- Triage complete

**Comment by ash1eygrace** [comment]
## Second Triage

I can't reproduce this anymore when adding a primary menu. Additionally, I checked the DOM after adding a primary navigation, and the navigation currently has the aria-labels set on the Main Nav, submenus, and the current page: 

```
<nav id="site-navigation" class="main-navigation" aria-label="Main Navigation">

<div class="menu-primary-container"><ul id="menu-primary" class="main-menu" aria-label="submenu">

<a href="https://mysite.wordpress.com/" aria-current="page">Home</a></nav>
```

Additionally, Google Page Speed shows: 

![Image](https://github.com/user-attachments/assets/5bc807e0-043d-4ba1-a677-7d7cd1b51298)

Going to close this as it's no longer an issue.
